examples/owin/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Microsoft.Owin.Hosting;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Owin;
+
+namespace Examples.Owin
+{
+    internal static class Program
+    {
+        public static void Main()
+        {
+            using var host = WebApp.Start(
+                "http://localhost:9000",
+                appBuilder =>
+                {
+                    appBuilder.UseOpenTelemetry();
```

Comment:
Maybe add a comment in the example guiding people to be sure to add this middleware first assuming they want to time the entire middleware pipeline.

Reply:
I added a comment for anyone who skips the README.